node-repository/src/main/java/com/yahoo/vespa/hosted/provision/restapi/v2/NodesResponse.java

```diff
@@ -212,6 +214,14 @@ private void toSlime(History history, Cursor array) {
         }
     }
 
+    private void toSlime(NodeResources resources, Cursor object) {
+        object.setDouble("vcpu", resources.vcpu());
+        object.setDouble("memoryGb", resources.memoryGb());
+        object.setDouble("diskGb", resources.diskGb());
+        object.setDouble("bandwithGbps", resources.bandwidthGbps());
```

Comment:
There's still a typo here.. :laughing:

Reply:
You want absolutely no typos in the entire word? High bar :-(